crates/ra_syntax/src/parsing/reparsing.rs

```diff
@@ -97,6 +102,9 @@ fn reparse_block<'node>(
 fn get_text_after_edit(element: SyntaxElement, edit: &AtomTextEdit) -> String {
     let edit =
         AtomTextEdit::replace(edit.delete - element.text_range().start(), edit.insert.clone());
+
+    // Note: we could move this match to a method or even further: use enum_dispatch crate
+    // https://crates.io/crates/enum_dispatch
```

Comment:
This is my proposal for the future refactoring, want to hear your opinion on that.

Reply:
I think there's some consensus that we already have a bit too many dependencies, so adding another one, especially a procedural macro, is probably not a good thing if we want to keep build times down.

Reply:
Okay, guys, that you for the opinion)